### ISSUE TITLE
why empty content?

### ISSUE THREAD

**Issue by one7two99** [Qubes-Community/Qubes-NAS#1]
Lets move this idea together with others into one page - keeping this place lean and thereof agile.

**Comment by taradiddles** [comment]
Yes, this should be moved in an idea page, together with qubes-AI.; my opinion is that we shouldn't even host projects (see the related "Workflow/Repo Summary " discussion in the admin team). @awokd and I are waiting for @Aekez 's feedback but it's been a week and a half without news.
